src/test/java/org/apache/commons/lang3/builder/ToStringBuilderTest.java

```diff
@@ -1041,7 +1041,7 @@ public void test_setUpToClass_invalid() {
         static final int staticInt2 = 67890;
     }
 
-    @Test
+    @Test(expected=IllegalArgumentException.class)
```

Comment:
This change is correct. The JavaDoc of the toString method says it throws an IllegalArgumentException for null inputs, but apparently it doesn't.